### ISSUE TITLE
Trigger function - iterative improvement and data collection for epi modelling

### ISSUE THREAD

**Issue by vollmersj** [corona-warn-app/cwa-documentation#264]
Will it be possible to extract information on false alarms to improve the trigger function - this could be improved through a/b testing.

In the same way it would future epidemic preparedness to know how a virus would spread for a prescribed trigger function - this is essentially the idea of [https://safeblues.org/](https://safeblues.org/) which could be easily integrated into the app.

**Comment by MikeJayDee** [comment]
How do you it was a false alarm? Even if you test, false negative rate of PCR tests is quite high, probably around 20%. And we would expect most that get an alarm not to be infected. I believe in manual contact tracing, around 90% of those quarantined never test negative.

I think this would need to be collected at the local health authorities. They will note down the reason for the test ("warning of CWA") and can then collect statistics.

**Comment by vollmersj** [comment]
There is clearly a trade off casting the net (time and space) too wide and on population level a lot of the test that are triggered by the app are negative.

**Comment by tkowark** [comment]
@vollmersj could you please make your idea a bit clearer? What do you mean by "trigger function"? 

Taking an educated guess, you're proposing the following:

* Have multiple ways of calculating "infection risk" and roll this out in a/b fashion to users
* Measure how many people have been tested after receiving an alert through the app
* Measure how many tested positive vs. negative to know which calculation is more accurate

Is that roughly the idea?

**Comment by vollmersj** [comment]
Yes that is roughly the idea. At the moment this seems to modelled in [transmission_risk.pdf](https://github.com/corona-warn-app/cwa-documentation/blob/master/transmission_risk.pdf). However,    it is not clear if one should be more conservative in proximity and duration of contact or if this leads to too many warnings. I guess  the current testing capacity [is not fully utilised.](https://www.tagesschau.de/inland/corona-tests-119.html) so there is a strong argument to be very conservative but capturing this data driven would be very useful and might become key if there is a second wave.  

The second idea is that start with a known trigger function/distribution and simulate spread of the virus using real contacts that is the idea of  (https://safeblues.org/)[https://safeblues.org/].

**Comment by tkowark** [comment]
Dear @vollmersj ,

thank you very much for your proposal.

To give the community a space for discussing new feature ideas or enhancements that might not be taken up immediately by the development teams, we created the cwa-wishlist repository.

This idea is one of these feature ideas or enhancements. Consequently, we will move this issue to this repository and allow the community and you to further refine it, discuss pros and cons, and evaluate alternatives.

The issue will not be closed in this repository to ensure long-term visibility. All issues in the cwa-wishlist repository will be discussed with our contracting entities. They will decide whether they will be implemented or pursued further. As soon as we have any updates, we will let you know about the details.

Best regards,
TK
Corona-Warn-App Open Source Team
